tests/foreman/ui/test_settings.py

```diff
@@ -175,6 +175,47 @@ def test_negative_validate_error_message(session, set_original_property_value):
                 assert is_valid_error_message(str(context.value))
 
 
+@tier2
+def test_positive_input_textarea_large(session):
+    """"Add values to large text box in setting
+
+    :id: 529ddd3a-1271-4043-9006-eac436b08b11
+
+    :expectedresults: Successfully add value to large text box
+
+    :CaseImportance: High
+    """
+    input = f'[ {gen_string("alpha")} ]'
+    property_name = 'host_dmi_uuid_duplicates'
+    with session:
+        session.settings.update(
```

Comment:
you are updating the property in settings, could create an adverse effect for other tests. please try to use https://github.com/SatelliteQE/robottelo/pull/7502/files#diff-3c2a24bcd6b2d5a25109128adda04e33R168 so you can restore the previous value.

Reply:
> you are updating the property in settings, could create an adverse effect for other tests. (because of setting is universal to the project) please try to use https://github.com/SatelliteQE/robottelo/pull/7502/files#diff-3c2a24bcd6b2d5a25109128adda04e33R168 so you can restore the previous value.

Reply:
I've implemented the reset of each changes within the test.